attic/test_product_gas.tz

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: 2021 Arthur Breitman
+# SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
```

Comment:
I think we'd better put this in a separate `.license` file and not change its content

Reply:
Added separate license file.